packages/transaction-controller/src/types.ts

```diff
@@ -100,6 +101,38 @@ export interface Transaction {
   estimateGasError?: string;
 }
 
+/**
+ * @type TransactionReceipt
+ *
+ * TransactionReceipt representation
+ * @property effectiveGasPrice - Effective gas price the transaction was charged at
+ * @property gasUsed - Gas used in the transaction
+ * @property l1Fee - Total used gas in hex
+ * @property status - The status of the transaction
+ */
+export interface TransactionReceipt {
+  blockHash?: string;
+  blockNumber?: number;
+  effectiveGasPrice?: string;
+  gasUsed?: string;
+  l1Fee?: string;
```

Comment:
I couldn't find a reference on the `l1Fee` but it's being used in transaction-breakdown component [here](https://github.com/MetaMask/metamask-extension/blob/6b72316eb68485ff4548275668646ea667c586ac/ui/components/app/transaction-breakdown/transaction-breakdown.container.js#L20).